configurations/html5.template.php

```diff
@@ -28,6 +28,10 @@
 $wgKalturaCDNUrl = wgGetUrl('cdn_host');
 // Default Stats URL
 $wgKalturaStatsServiceUrl = wgGetUrl('stats_host');
+// Default Live Stats URL
+$wgKalturaLiveStatsServiceUrl = wgGetUrl('live_stats_host');
+// Default Kaltura Analytics URL
+$wgKalturaAnalyticsServiceUrl = wgGetUrl('analytics_host');
 
 // SSL host names
 if( $wgHTTPProtocol == 'https' ){
```

Comment:
isn't that what's done 3 lines below?
https://github.com/kaltura/server/blob/42201ef841cd5c0ff98d4dc3d98c182576a19a29/configurations/html5.template.php#L37